doc/neps/ufunc-overrides.rst

```diff
@@ -274,24 +314,24 @@ Here is a demo highlighting the functionality.::
     In [6]: np.multiply(a, b)
     Out[6]: 'B'
 
-A simple ``__numpy_ufunc__`` has been added to SciPy's sparse matrices
-Currently this only handles ``np.dot`` and ``np.multiply`` because it was the 
-two most common cases where users would attempt to use sparse matrices with ufuncs.
-The method is defined below::
+As a simple example, one could add the following ``__array_ufunc__`` to
+SciPy's sparse matrices (just for ``np.dot`` and ``np.multiply`` as
+these are the two most common cases where users would attempt to use
+sparse matrices with ufuncs)::
 
-    def __numpy_ufunc__(self, func, method, pos, inputs, **kwargs):
+    def __array_ufunc__(self, func, method, pos, inputs, **kwargs):
```

Comment:
Not also that the arguments are "normalized" before the call to `__array_ufunc__` so that only the input arrays are in `inputs`, everything else is put into `kwargs`.